frontend/src/Response/Table/FixedTable.tsx

```diff
@@ -0,0 +1,27 @@
+import { QuestionInterface } from "../Question";
+import InputInfo, { InputInfoInterface } from "../SimpleQuestions/InputInfo";
+import { WithInputInfoInterface } from "../SimpleQuestions/LabeledQuestion";
+import SimpleQuestion, { SimpleQuestionInterface } from "../SimpleQuestions/SimpleQuestion";
+import BaseTable from "./BaseTable";
+
+export interface FixedTableInterface {
+    inputInfoOnTop: Array<InputInfoInterface>;
+    inputInfoOnLeft: Array<InputInfoInterface>;
+    questionData: SimpleQuestionInterface;
+}
+
+interface FixedTableWithInputInfoInterface extends FixedTableInterface, WithInputInfoInterface {}
+
+function FixedTable(fixedTableData: FixedTableWithInputInfoInterface): JSX.Element {
```

Comment:
![image](https://user-images.githubusercontent.com/47260215/217600830-f51d0215-582e-46ec-a072-92d8650bf27d.png)

I added some more columns and table didn't change it's size despite that it had some more place in the right.
I think it would be better to make this table change width in depending of number of columns and make it scrollable if it's too thick for the page

Reply:
Thanks, fixed